docs/alma/alma.rst

```diff
@@ -124,47 +123,37 @@ Querying Targets and Regions
 You can query by object name or by circular region:
 
 .. code-block:: python
-
+.. doctest-remote-data::
     >>> from astroquery.alma import Alma
     >>> m83_data = Alma.query_object('M83')
```

Comment:
interestingly I run into an error here. It's certainly unrelated to the PR, but would hold back merge until it's resolved.

Reply:
When the tests are not run together with the rest of the module's test suite, then it's more sensible, and relevant failures are found. So let's keep running `pytest docs/alma/alma.rst --remote-data` for this PR.

Reply:
this query works fine for me right now and produces 400 lines of talbe, so I consider this resolved.

(but yes let's keep running the tests...)